godtools/ViewControllers/Tract/TractViewController.swift

```diff
@@ -76,10 +76,8 @@ class TractViewController: BaseViewController {
     }
 
     override func displayScreenTitle() {
-        if parallelLanguageIsAvailable() {
+        if parallelLanguageIsAvailable() && determinePrimaryLabel() != determineParallelLabel() {
```

Comment:
Ah, yes I want to do a small fix to make sure that the user cannot choose the same language for primary and parallel. Then we don't have to do this in the future. This works for now though. 👍